tests/alembic/test_alembic.py

```diff
@@ -13,6 +13,7 @@
 from invenio_db.utils import alembic_test_context, drop_alembic_version_table
 
 
+@pytest.mark.skip(reason="We have to figure out why this is failing (or even if it is failing).")
```

Comment:
i tried the tests without the skip it worked for me locally, could you please try to remove it so we can see if the tests on github will work?

Reply:
:+1: Tests are working now!